.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+          cache: 'npm'
+      - name: Install
+        run: npm install
```

Comment:
I think we probably want to use `npm ci` here instead of `npm install`

Reply:
nice! didn't know about this